novawallet/Modules/NetworkManagement/Traits/CustomNetworkSetupTrait.swift

```diff
@@ -0,0 +1,279 @@
+import Foundation
+import SubstrateSdk
+import Operation_iOS
+
+protocol CustomNetworkSetupTrait: NetworkNodeCorrespondingTrait {
```

Comment:
I think a separate operation factory suits better here then trait as we create an operation that needs a lot of parameters. I would initialise the factory with parameters and have I single createWrapper() method

Reply:
Agree, will do it as a separate PR